Comparison Engine/Comparison Engine/GoogleMap/MapController.cs

```diff
@@ -26,8 +26,9 @@ private MapController()
         }
 
         // set default values so it would compile, but will need to set these depending on user or from current location
-        public double prefDistance { private get; set; } = 69; 
-        public string homeAddress { private get; set; } = "S. Nėries 99";
+        public double prefDistance { get; set; } = 69; // user's preferred distance to a bar
+        public string homeAddress { get; set; } = "S. Nėries 99"; // user's current location (for now home location until we can just get current location)
+        public bool prefWalking { get; set; } = true; // user's preference to walk/drive
```

Comment:
Something tells me that if we're making an app for people to find bars it's kind of irresponsible to let the user use drive as a way to get to a bar. Kinda seems like we're encouraging drunk driving

Reply:
Also, if we take this out, we lose optional arguments usage, because there won't be a point to make two different calls in ShowRoute depending on whether walking mode is on or off

Reply:
Fair point